internal/slicer/slicer.go

```diff
@@ -330,3 +339,15 @@ func contains(l []string, s string) bool {
 	}
 	return false
 }
+
+func isListedInSlice(extractInfo *deb.ExtractInfo, slice *setup.Slice) bool {
+	if extractInfo == nil {
+		return false
+	}
```

Comment:
Following up on previous comment, we can probably ditch this check from this function.